contracts/Super1155.sol

```diff
@@ -1005,4 +951,24 @@ contract Super1155 is PermitControl, ERC165Storage, IERC1155, IERC1155MetadataUR
     locked = true;
     emit CollectionLocked(_msgSender());
   }
+
+
+  function getSupplyType(uint256 _groupId) public view returns (DFStorage.SupplyType) {
```

Comment:
I'm not really a fan of getters for specific properties when the underlying storage is public; I'd rather not have this or `getItemType`, `getBurnType` or `getGroupTypes`.